crates/solvers/src/infra/config/legacy.rs

```diff
@@ -25,6 +25,9 @@ struct Config {
 
     /// The URL of the endpoint that responds to solve requests.
     endpoint: String,
+
+    /// Enabled requests compression
+    gzip_requests: bool,
```

Comment:
nit: Probably makes sense to use `#[serde(default)]` here to not require everybody to set this argument since it's just configuring an optional optimization.